src/cascadia/TerminalApp/TelnetGenerator.h

```diff
@@ -0,0 +1,19 @@
+/*++
+Copyright (c) Microsoft Corporation
+Licensed under the MIT license.
+
+Module Name:
+- TelnetGenerator
+
+Abstract:
+- Information needed to detect a Telnet connection type.
+
+Author(s):
+- Michael Niksa - 2019-12-05
+
+--*/
+
+#pragma once
+
+// {311153fb-d3f0-4ac6-b920-038de7cf5289}
+static constexpr GUID TelnetConnectionType = { 0x311153fb, 0xd3f0, 0x4ac6, { 0xb9, 0x20, 0x03, 0x8d, 0xe7, 0xcf, 0x52, 0x89 } };

```

Comment:
The last idea I had was "this isn't really necessary at all, we could just detect the protocol in the command line string", but I don't want to get in the way of things right now. We should do that as a follow on and maybe just call everything conpty://, azure://, telnet://, etc.

Reply:
Sounds like you should file an issue ;)

Reply:
NAK: i think URIs are the worst form of communication

Reply:
OK.